### PR TITLE
Add links to blog post

### PR DESCRIPTION
**Pull request by sadikoff**
closes SymfonyCasts/tutorial-issues#1

### CODE REVIEW

File: knpu/behat-hooks-background.md [hunk 1]
```diff
@@ -94,6 +94,11 @@ a hold of. To get access to them we just need to add a require statement in our
 
 [[[ code('16793205d0') ]]]
 
+***TIP
+That's cool, but unnecessary with Symfony 5. Read more about Behat + phpunit configuration
+[here](https://symfonycasts.com/blog/behat-symfony#configure-it)
+***
```
Comment (bocharsky-bw) [inline knpu/behat-hooks-background.md:100]: > This is a redundant step if you use Symfony 5. Read more about how to configure Behat & PHPUnit properly with Symfony 5 [here](https://symfonycasts.com/blog/behat-symfony#configure-it).

File: knpu/clear-data-symfony-extension.md [hunk 2]
```diff
@@ -66,6 +66,11 @@ Perfect! We can run this over and over because it's clearing out the data first.
 And, surprise! There's an easier way to bootstrap Symfony and clear out the database.
 I always like taking the long way first so we can see how things work.
  
+***TIP
+Working with Symfony 5? Check out this [article](https://symfonycasts.com/blog/behat-symfony#install-behat-with-everything-you-need)
```
Comment (bocharsky-bw) [inline knpu/clear-data-symfony-extension.md]:
```suggestion
Working with Symfony 5? Check out this [blog post](https://symfonycasts.com/blog/behat-symfony#install-behat-with-everything-you-need)
```

File: knpu/install.md [hunk 3]
```diff
@@ -42,6 +42,10 @@ Hold onto your butts, let's going to get this thing running!
 
 ## Install and Configuration
 
+***TIP
+Installing Behat on Symfony 5 or newer? Read this article: [Setting up Behat on Symfony 5](https://symfonycasts.com/blog/behat-symfony)
+***
```
Comment (bocharsky-bw) [inline knpu/install.md:48]: > Installing Behat on Symfony 5 or newer? Read this [blog post](https://symfonycasts.com/blog/behat-symfony) to know about how to install and configure it properly.

File: knpu/install.md [hunk 4]
```diff
@@ -125,6 +129,12 @@ isn't going to work since this isn't actually opening up a real browser. So, how
 this in a real browser? There are actually a bunch of different ways. The easiest is by
 using Selenium. 
 
+***TIP
+With Symfony 5 it became easier! You can find required instructions about Selenium configuration
+[here](https://symfonycasts.com/blog/behat-symfony#real-browser-power) or try [Panther](https://symfonycasts.com/blog/behat-panther).
+working with Panther is great and a bit easier than Selenium.
+***
```
Comment (bocharsky-bw) [inline knpu/install.md:138]: > With Symfony 5 it became easier! You can find the required instructions about Selenium configuration [here](https://symfonycasts.com/blog/behat-symfony#real-browser-power). Or you can try a new library called [Panther](https://symfonycasts.com/blog/behat-panther) that is a bit easier than Selenium, but we're going to cover Selenium further in this course.

**Review by bocharsky-bw — COMMENTED** [review]
I left some comments about how to improve the notes. Though, I have some doubts if we should link to the article or probably would be better to add all required steps in the notes. I'm afraid that newcomers may be lost in links. What if we just will copy/paste from the blog post everything users need at the specific spots? It will make it easier to follow the tutorial and do not switch to the blog post when you follow it. Thoguhts?

**Comment by bocharsky-bw** [comment]
@sadikoff What do you think about my comment: https://github.com/knpuniversity/behat/pull/43#pullrequestreview-766300629

Should we talk about it today at the meeting?

**Comment by sadikoff** [comment]
@bocharsky-bw sorry I forgot about this issue again =) I applied your variant to the text. I don't think that it will be better to make a tip for every step that differs, even some code blocks may be affected. And honestly, I think that it will be really better to record another course and cover Panther and its usage.

**Comment by bocharsky-bw** [comment]
Thank you for taking care of this! This should be very useful for users until we re-record this tutorial :)

**Pull request merged**